content/github/getting-started-with-github/create-a-repo.md

```diff
@@ -17,7 +17,7 @@ topics:
 
 {% if currentVersion == "free-pro-team@latest" %}
 
-You can store a variety of projects in {% data variables.product.product_name %} repositories, including open source projects. With [open source projects](http://opensource.org/about), you can share code to make better, more reliable software.
+You can store a variety of projects in {% data variables.product.product_name %} repositories, including open source projects. With [open source projects](http://opensource.org/about), you can share code to make better, more reliable software. Also, there are various types of repositories for different purposes about which you can read more on [About Repositories](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-repositories).
```

Comment:
"Various types of repositories" and "different purposes" is a bit vague. Let's be a bit more specific (but still concise!) about the information folks will find in the link.

We should also use relative links, formatted like this.

```suggestion
You can store a variety of projects in {% data variables.product.product_name %} repositories, including open source projects. With [open source projects](http://opensource.org/about), you can share code to make better, more reliable software. You can use repositories to collaborate with others and track your work. For more information, see "[About repositories](/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-repositories)."
```